cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -368,15 +380,16 @@ script "js_databricks_workspaces_filtered", type: "javascript" do
   code <<-EOS
   var results = []
   if ( param_databricks_workspace_list.length != 0){
-    results = []
     _.each(param_databricks_workspace_list, function(workspace){
-      var found = _.find(ds_databricks_workspaces, function(item){
-        return (item.workspaceId == workspace || item.name.toLowerCase() == workspace.toLowerCase());
+      getIterator(ds_databricks_workspaces).Each(function(item, idx) {
+        if (item.workspaceId == workspace || item.name.toLowerCase() == workspace.toLowerCase()) {
+          appendToResult(item);
+          return true;
+        }
       })
-      results.push(found)
     })
   } else {
-    results = ds_databricks_workspaces
+    getIterator(ds_databricks_workspaces).Each(appendToResult)
   }
 EOS
 end
```

Comment:
<blockquote>
Possible invalid spacing between comma-separated items found:

Line 447: end_date = end_date.toISOString().split("T")[0].split("-").slice(0,2).join("-")

Line 451: start_date = start_date.toISOString().split("T")[0].split("-").slice(0,2).join("-")

Line 602: "aggregation": "minimum,average,maximum",

Line 605: "metricnames": "Percentage CPU,Available Memory Bytes",

Line 726: var measurements = _.compact(_.pluck(resourceMetrics.data,stat))

Line 1032: "virtualmachines": result_map[subscriptionId][location]["virtualmachines"].slice(i,i+chunk_size)

Line 1108: "Delete Databricks Cluster",vm["databricks_cluster_id"],

Line 1110: "in Azure Subscription", vm["databricks_workspace"]["subscriptionName"],"(", vm["databricks_workspace"]["subscriptionId"], ")"

Line 1125: "Resize Databricks Cluster",vm["databricks_cluster_id"],

Line 1127: "in Azure Subscription", vm["databricks_workspace"]["subscriptionName"],"(", vm["databricks_workspace"]["subscriptionId"], ")",

Line 1128: "from", vm["resourceType"],"to", vm["newResourceType"]

Line 1192: check logic_or($ds_parent_policy_terminated,    eq(0,size(data))    )

Line 1353: check logic_or($ds_parent_policy_terminated,    eq(0,size(data))    )

Line 1530: path join(["/api/governance/projects/", rs_project_id, "/applied_policies/", switch(ne(meta_parent_policy_id,""), meta_parent_policy_id, policy_id) ])

Comma separated items should be organized as follows, with a single space following each comma: apple, banana, pear
</blockquote>